### PR TITLE
Add `App` and `ContentType` CRUD

### PR DESCRIPTION
**Pull request by TheHadiAhmadi**

### CODE REVIEW

File: src/FluentCMS.Web.UI.Components/Components/Form/Form.razor.cs
Comment (ParsaGachkar) [inline src/FluentCMS.Web.UI.Components/Components/Form/Form.razor.cs:1]: @babakhani i think this file is getting out of control. let's break it into multiple files and it looks like to be a separate task on its own

File: src/FluentCMS.Web.UI.Components/Components/Form/Select.razor
Comment (ParsaGachkar) [inline src/FluentCMS.Web.UI.Components/Components/Form/Select.razor:1]: same here, let's keep these issues on separate pr's

File: src/FluentCMS.Web.UI/Helper.cs
Comment (ParsaGachkar) [inline src/FluentCMS.Web.UI/Helper.cs:1]: i think this can be generalised into a single method to avoid code duplication, again not related to this issue

File: src/FluentCMS.Web.UI.Components/Components/Form/Select.razor
Comment (TheHadiAhmadi) [reply src/FluentCMS.Web.UI.Components/Components/Form/Select.razor:1]: I needed `OnChange` for `Select` component, for App Selection dropdown.
We can add OnChange to other form elements in Separate PR.

**Review by babakhani — APPROVED** [review]
